modules/ppcp-settings/services.php

```diff
@@ -573,6 +575,7 @@
 			'acdc'                        => $features['advanced_credit_and_debit_cards']['enabled'] ?? false,
 			'save_paypal'                 => $features['save_paypal_and_venmo']['enabled'] ?? false,
 			'alternative_payment_methods' => $features['alternative_payment_methods']['enabled'] ?? false,
+			'installments' => $features['installments']['enabled'] ?? false,
```

Comment:
I wonder why the PHPcs is not failing here, cause it looks like this is not aligned correctly.